### PR TITLE
Rich/features/contextual footer

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?

- Lint and clean up ```contextual-footer.scss```
- Improve the markup in ```index.html```

#### Where should the reviewer start?

Run ```gulp test``` & ```gulp build```

#### How should this be manually tested?

Open a view ```index.html``` for any visual issues

#### Any background context you want to provide?

_n/a_

#### What are the relevant tickets?

_n/a_

#### Screenshots (if appropriate)

_n/a_

#### Questions:

Do migrations need to be ran? _n/a_

- Are there any additional deployment instructions/requirements? _n/a_

### CODE REVIEW

**Comment by anthonydillon** [comment]
4 issues found in ../scss/patterns/_contextual-footer.scss

File: index.html [hunk 1]
```diff
@@ -45,203 +45,203 @@
...
-        </div>
-        <div class="twelve-col">
-            <h2>Buttons</h2>
-            <div class="row">
-                <div class="six-col">
-                    <button class="cta-ubuntu" href="#">Button</button>
+        <div id="main-content" class="inner-wrapper">
+            <div class="row">
+                <h2>Grid</h2>
+                <div class="twelve-col box">
```
Comment (anthonydillon) [inline index.html]: Can you remove this div please.

File: index.html [hunk 2]
```diff
@@ -250,212 +250,210 @@
...
+                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Libero temporibus, cupiditate harum fugiat dignissimos! Quo accusantium dolorum, deleniti qui laudantium excepturi dolore voluptas obcaecati amet.</p>
+                        </article>
                     </div>
                 </div>
-                <div class="four-col last-col">
-                    <h4>.box-orange</h4>
-                    <div class="box box-orange four-col">
-                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt laudantium saepe provident facilis nostrum aspernatur, itaque reiciendis ea consectetur necessitatibus minima, pariatur dolorum rem repellat?</p>
+                <div class="twelve-col">
+                    <div class="twelve-col">
```
Comment (anthonydillon) [inline index.html]: wft?

**Comment by richmccartney** [comment]
@anthonydillon Can you take another look now.

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**